src/denostore.ts

```diff
@@ -1,15 +1,22 @@
 import { Router } from 'https://deno.land/x/oak@v10.2.0/mod.ts';
 import { renderPlaygroundPage } from 'https://deno.land/x/oak_graphql@0.6.3/graphql-playground-html/render-playground-html.ts';
 import { graphql } from 'https://deno.land/x/graphql_deno@v15.0.0/mod.ts';
-import { DenostoreArgs } from './types.ts';
 import { queryParser } from './utils.ts';
 
+import type {
+  Redis,
+  GraphQLSchema,
+  DenostoreArgs,
+  Middleware,
+  Context,
+} from './types.ts';
+
 export default class Denostore {
-  #schema: any;
-  #usePlayground?: boolean;
-  #redisClient: any;
-  #router: any;
-  #route?: string;
+  #usePlayground: boolean;
```

Comment:
should be optional right?

Reply:
and I meant usePlayground in this comment.

Reply:
yes